src/ui_iteminfo.cpp

```diff
@@ -0,0 +1,153 @@
+                if( i.sName == "--" ) {
+                    ImGui::Separator();
+                } else {
+                    window.draw_colored_text( i.sName, base_color, wrap_width );
+                }
+            }
+        } else {
+            if( same_line ) {
+                ImGui::SameLine( 0, 0 );
+            }
+            std::string formatted_string;
+            if( i.bDrawName ) {
+                formatted_string = i.sName;
+            }
+
+            std::string sFmt = i.sFmt;
+
+            //A bit tricky, find %d and split the string
+            size_t pos = sFmt.find( "<num>" );
+            if( pos != std::string::npos ) {
+                formatted_string += sFmt.substr( 0, pos );
+            } else {
+                formatted_string += sFmt;
+            }
+
+            if( i.sValue != "-999" ) {
+                nc_color this_color = c_yellow;
+                for( const iteminfo &k : data.get_item_compare() ) {
+                    if( k.sValue != "-999" ) {
+                        if( i.sName == k.sName && i.sType == k.sType ) {
+                            double iVal = i.dValue;
+                            double kVal = k.dValue;
+                            if( i.sFmt != k.sFmt ) {
+                                // Different units, compare unit adjusted vals
+                                iVal = i.dUnitAdjustedVal;
+                                kVal = k.dUnitAdjustedVal;
+                            }
+                            if( iVal > kVal - .01 &&
+                                iVal < kVal + .01 ) {
+                                this_color = c_light_gray;
+                            } else if( iVal > kVal ) {
+                                if( i.bLowerIsBetter ) {
+                                    this_color = c_light_red;
+                                } else {
+                                    this_color = c_light_green;
+                                }
+                            } else if( iVal < kVal ) {
+                                if( i.bLowerIsBetter ) {
+                                    this_color = c_light_green;
+                                } else {
+                                    this_color = c_light_red;
+                                }
+                            }
+                            break;
+                        }
+                    }
+                }
+                formatted_string += colorize( i.sValue, this_color );
+            }
+            if( pos != std::string::npos ) {
+                formatted_string += sFmt.substr( pos + 5 );
+            }
+            window.draw_colored_text( formatted_string, base_color, wrap_width );
+            same_line = !i.bNewLine;
+        }
+    }
+}
+
+iteminfo_window::iteminfo_window( item_info_data &info, point pos, int width, int height,
+                                  ImGuiWindowFlags flags ) :
+    cataimgui::window( "iteminfo", flags ),
```

Comment:
While functional, this title leaves something to be desired. And it should be localizable. Might I suggest `_("Item info")`?

Reply:
Oops, that's just the working title I forgot to update.